.github/workflows/c-cpp.yml

```diff
@@ -91,7 +91,7 @@ jobs:
               cc: "clang",
               cxx: "clang++",
               generators: "Xcode",
-              cmake_args: "-DLVK_WITH_TRACY=OFF"
+              cmake_args: "-DLVK_WITH_TRACY=OFF -DLVK_WITH_VULKAN_PORTABILITY=ON"
```

Comment:
Let's turn this on by default.

Reply:
Let's hide the defaults selection logic inside `CMakeLists.txt`. The goal is to keep the command line shorter.